react/features/deep-linking/functions.js

```diff
@@ -11,6 +11,44 @@ import {
 import { _shouldShowDeepLinkingDesktopPage }
     from './shouldShowDeepLinkingDesktopPage';
 
+/**
+ * Indicates whether the window load event was already received.
+ *
+ * @type {boolean}
+ */
+let windowIsLoaded = false;
+
+/**
+ * Handler for the window load event.
+ *
+ * @returns {void}
+ */
+function onWindowLoad() {
+    windowIsLoaded = true;
```

Comment:
Is window.document.readyState an equivalent value?

Reply:
When I'm reading the description on MDN they sound the same but it turns out that in reality they are not! I was testing it because if they were equivalent it would be so much easier to implement.